layouts/partials/hero/background.html

```diff
@@ -4,12 +4,17 @@
 {{- $featured := $images.GetMatch "*background*" -}}
 {{- if not $featured }}{{ $featured = $images.GetMatch "*feature*" }}{{ end -}}
 {{- if not $featured }}{{ $featured = $images.GetMatch "{*cover*,*thumbnail*}" }}{{ end -}}
-{{ if and .Params.featureimage (not $featured) }}
-{{- $url:= .Params.featureimage -}}
-{{ $featured = resources.GetRemote $url }}
+{{ $isParentList := eq (.Scratch.Get "scope") "list"  }}
+{{ $url:= default nil }}
+{{ if $isParentList }}
+{{ $url = default $.Site.Params.homepage.homepageImage $.Site.Params.defaultBackgroundImage }}
+{{ else }}
+{{ $url = .Params.featureimage }}
+{{ if not $url }} {{ $url = default $.Site.Params.homepage.homepageImage $.Site.Params.defaultBackgroundImage }} {{ end }}
 {{ end }}
+{{- if not $featured }}{{ $featured = resources.GetRemote $url }}{{ end -}}
 {{- if not $featured }}{{ with .Site.Params.defaultBackgroundImage }}{{ $featured = resources.Get . }}{{ end }}{{ end -}}
```

Comment:
Here is what these lines mean in plain English.

If the scope is `list` (that is list view) then set the URL as `homepageImage` or `defaultBackgroundImage`

else when it is article view set URL as `featureimage` if featureimage is not present set article background image as `homepageImage` or `defaultBackgroundImage`